tests/robustness/makefile.mk

```diff
@@ -1,6 +1,6 @@
 .PHONY: test-robustness-reports
 test-robustness-reports:
-	cd ./tests && go test ./robustness/validate -v --run TestDataReports
+	cd ./tests && go test ./robustness/validate -v --count 1 --run TestDataReports
```

Comment:
Is 1 the default value for `--count`?

Reply:
No, https://github.com/golang/go/issues/24573

Reply:
To be clearer, the default value for `--count` is 1 for sure. But there's a difference between providing it with the default value, or not providing it at all.  Cache will be disabled in the first case.